.travis.yml

```diff
@@ -171,10 +171,15 @@ before_install:
         fi
   - virtualenv --version
   - python --version
+  - |
+        # upgrade setuptools if it's too old to support PEP 508
+        # PEP 508 support is needed for python version dependent dependencies in requirements.txt/setup.py (e.g. futures)
+        pip install "setuptools>=20.2"
+  - pip list
```

Comment:
we already have `pip freeze` in `before_script`. We don't really install anything until this point (except maybe setuptools), I can't see what good does putting pip list here give us. Could you please help me understand it (or remove the line)?